scss/_tags.scss

```diff
@@ -0,0 +1,159 @@
+// stylelint-disable function-disallowed-list
+
+.tag {
+  // scss-docs-start tag-css-vars
+  --#{$prefix}tag-gap: #{$tag-gap};
+  --#{$prefix}tag-padding-x: #{$tag-padding-x};
+  --#{$prefix}tag-padding-y: #{$tag-padding-y};
+  --#{$prefix}tag-margin: 0;
+  @include rfs($tag-font-size, --#{$prefix}tag-font-size);
+  --#{$prefix}tag-font-shift: #{$tag-font-shift};
+  --#{$prefix}tag-font-weight: #{$tag-font-weight};
+  --#{$prefix}tag-font-spacing: #{$tag-font-spacing};
```

Comment:
This is not the default font spacing?

Reply:
It is indeed, but if we propose to change the font-size, we should propose to change the font-spacing IMO.